### PR TITLE
Tools: Link to .NET Standard docs

### LINKED ISSUE #12608: dotnet ef dbcontext scaffold can't be run on .NETStandard class library projects
Running the following on a .NET Standard Class Library:

```bash
dotnet ef dbcontext scaffold <CONNECTION> Microsoft.EntityFrameworkCore.SQLServer
```
Will produce this error:

>Startup project 'projectName' targets framework '.NETStandard'. There is no runtime associated with this framework, and projects targeting it cannot be executed directly. To use the Entity Framework Core Package Manager Console Tools with this project, add an executable project targeting .NET Framework or .NET Core that references this project, and set it as the startup project; or, update this project to cross-target .NET Framework or .NET Core.

Similar issues on StackOverflow:

* [PowerShell](https://stackoverflow.com/questions/48673987/trying-to-set-up-entity-framework-core-in-net-standard-project)
* [dotnet cli](https://stackoverflow.com/questions/49218173/ef-core-database-first-in-class-library-using-visual-studio-code)
* [PowerShell again](https://stackoverflow.com/questions/48558951/issues-with-adding-entity-framework-to-net-standard-class-library)
* [Another](https://stackoverflow.com/questions/44430963/is-ef-core-add-migration-supported-from-net-standard-library)

Note: All those users are using the tools as expected. It used to work before Core if I remember correctly. 

### Steps to reproduce

Get your command line to a directory with a project of type .NET Standard Class Library and run the following command,

```bash
dotnet ef dbcontext scaffold <CONNECTION> Microsoft.EntityFrameworkCore.SQLServer
```

### Further technical details
EF Core version: 2.1.1
Database Provider: Microsoft.EntityFrameworkCore.SqlServer
Operating system: Windows 10
IDE: CLI

csproj content:

```xml
<Project Sdk="Microsoft.NET.Sdk">

  <PropertyGroup>
    <TargetFramework>netstandard2.0</TargetFramework>
    <ApplicationIcon />
    <StartupObject />
  </PropertyGroup>

  <ItemGroup>
    <PackageReference Include="Microsoft.EntityFrameworkCore.Design" Version="2.1.1" />
    <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Version="2.1.1" />
    <PackageReference Include="NuGet.Protocol.Core.Types" Version="4.2.0" />
    <PackageReference Include="NuGet.Protocol.Core.v3" Version="4.2.0" />
    <PackageReference Include="Semver" Version="2.0.4" />
  </ItemGroup>

</Project>
```
// @spboyer for tracking

### PR DESCRIPTION
**Pull request by bricelam**
Fixes #12608

### CODE REVIEW

File: src/EFCore.Tools/tools/EntityFrameworkCore.psm1 [hunk 1]
```diff
@@ -822,7 +822,8 @@ function EF($project, $startupProject, $params, [switch] $skipBuild)
             'runtime associated with this framework, and projects targeting it cannot be executed directly. To use ' +
             'the Entity Framework Core Package Manager Console Tools with this project, add an executable project ' +
             'targeting .NET Framework or .NET Core that references this project, and set it as the startup project; ' +
-            'or, update this project to cross-target .NET Framework or .NET Core.'
+            'or, update this project to cross-target .NET Framework or .NET Core. For more information on using the ' +
+            'EF Core Tools with .NET Standard projects, see https://docs.microsoft.com/en-us/ef/core/miscellaneous/cli/powershell#other-target-frameworks'
```
Comment (bricelam) [inline src/EFCore.Tools/tools/EntityFrameworkCore.psm1]: @divega Can I get a fwlink?

File: src/dotnet-ef/Properties/Resources.resx [hunk 2]
```diff
@@ -229,7 +229,7 @@
...
-    <value>Startup project '{startupProject}' targets framework '.NETStandard'. There is no runtime associated with this framework, and projects targeting it cannot be executed directly. To use the Entity Framework Core .NET Command-line Tools with this project, add an executable project targeting .NET Core or .NET Framework that references this project, and set it as the startup project using --startup-project; or, update this project to cross-target .NET Core or .NET Framework.</value>
+    <value>Startup project '{startupProject}' targets framework '.NETStandard'. There is no runtime associated with this framework, and projects targeting it cannot be executed directly. To use the Entity Framework Core .NET Command-line Tools with this project, add an executable project targeting .NET Core or .NET Framework that references this project, and set it as the startup project using --startup-project; or, update this project to cross-target .NET Core or .NET Framework. For more information on using the EF Core Tools with .NET Standard projects, see https://docs.microsoft.com/en-us/ef/core/miscellaneous/cli/dotnet#other-target-frameworks</value>
```
Comment (bricelam) [inline src/dotnet-ef/Properties/Resources.resx]: @divega Ditto. (Same section but in the CLI docs)

File: src/EFCore.Tools/tools/EntityFrameworkCore.psm1 (hunk 1 above)
Comment (divega) [reply src/EFCore.Tools/tools/EntityFrameworkCore.psm1]: @bricelam https://go.microsoft.com/fwlink/?linkid=2034705

File: src/dotnet-ef/Properties/Resources.resx (hunk 2 above)
Comment (divega) [reply src/dotnet-ef/Properties/Resources.resx]: @bricelam https://go.microsoft.com/fwlink/?linkid=2034781

File: src/EFCore.Tools/tools/EntityFrameworkCore.psm1 (hunk 1 above)
Comment (bricelam) [reply src/EFCore.Tools/tools/EntityFrameworkCore.psm1]: Thanks!

**Pull request merged**